src/GovUk.Education.ExploreEducationStatistics.Content.Services.Tests/PublicationServiceTests.cs

```diff
@@ -1537,7 +1537,7 @@ public async Task Search_SortByRelevance_Desc()
                 var pagedResult = (await service.ListPublications(
                     search: "term",
                     sort: null, // Sort should default to relevance
-                    order: null // Order should default to descending
+                    sortDirection: null // Order should default to descending
```

Comment:
Code comments like this one could be changed to `Direction should default to descending`. This applies to multiple places so worth checking elsewhere.

Reply:
Simplified this to 'Defaults to descending'. Similar thing applied to rest of the comments too